src/index.ts

```diff
@@ -61,6 +62,17 @@ export class Popgun {
     return popEngine.isPopAlreadyOpenForGroup(groupId);
   }
 
+  // reposition a pop based on the groupId
+  public reposition(groupId: string): void {
```

Comment:
Cool! If we ever need to support repositioning to a different target, I can see us being able to just extend this to take in a second optional parameter.